docs/docs/references/sql-runner.mdx

```diff
@@ -19,36 +26,56 @@ The SQL runner can be accessed from the `new` --> `Query using SQL runner` optio
   style={{ display: 'block', margin: '0 auto 20px auto' }}
 />
 
-Once you're in the SQL runner, you'll see two key components:
+Once you're in the SQL Runner, you'll see four key components:
 
-1. A list of tables that correspond to all of the models in your connected dbt project.
-2. A SQL query builder.
+1. A list of tables in your data warehouse that Lightdash has access to (based on your data warehouse connection in Lightdash).
+2. The schema of a selected table.
+3. The SQL query builder. 
+4. The results from your latest SQL query.
 
 <img
   src={SQLRunnerComponents}
-  width="981"
-  height="652"
+  width="1082"
+  height="591"
   style={{ display: 'block', margin: '0 auto 20px auto' }}
 />
 
-Only tables included in your dbt project are shown in the sidebar, but you can query any tables that your Lightdash data warehouse connection has access to (this is based on the data warehouse credentials that you setup in your project connection settings).
+To build a query in the SQL Runner, just write your query in the SQL query builder, then hit `run query` to see the results. 
 
-Once you've run your query, you'll see the query's results and a chart created in the `chart` panel.
+You can access your **query history** from your latest SQL Runner session by clicking on the `query history` button beside `run query`.
```

Comment:
i think `Query history` and maybe even `Layout` should be headers so they show up as page sections